tests/conftest.py

```diff
@@ -13,6 +13,9 @@
 from cachito.web.app import create_app
 
 
+os.environ["PROMETHEUS_MULTIPROC_DIR"] = "/tmp/wsgi"
```

Comment:
Please do cleanup for envvar change https://shay-palachy.medium.com/temp-environment-variables-for-pytest-7253230bd777

Reply:
Cleaned up.